proto/v1/release_service.proto

```diff
@@ -122,14 +129,43 @@ message UndeleteReleaseRequest {
   ];
 }
 
+message CheckReleaseRequest {
+  // Format: projects/{project}
+  string parent = 1 [
+    (google.api.field_behavior) = REQUIRED,
+    (google.api.resource_reference) = {type: "bytebase.com/Project"}
+  ];
+
+  // The release to check.
+  Release release = 2 [(google.api.field_behavior) = REQUIRED];
+
+  // The targets to dry-run the release.
+  // Can be database or databaseGroup.
+  // Format:
+  // projects/{project}/databaseGroups/{databaseGroup}
+  // instances/{instance}/databases/{database}
+  repeated string targets = 3;
+}
```

Comment:
Nit: I suspect we will take in SQL review configuration ID for checks without targets. But let's forget about it for now.